package.json

```diff
@@ -61,6 +61,7 @@
     "my-local-ip": "^1.0.0",
     "node-uuid": "^1.4.3",
     "passport": "^0.3.2",
+    "openid": "^0.5.13",
```

Comment:
well, I would prefer to use [passport-openid](https://github.com/jaredhanson/passport-openid) module, since other unit are using passport. For code consistency reason